src/Evolution/Systems/CurvedScalarWave/Worldtube/Tags.cpp

```diff
@@ -91,7 +92,7 @@ void FaceCoordinatesCompute<Dim, Frame, Centered>::function(
                   mesh.extents(), direction.value().dimension(),
                   index_to_slice_at(mesh.extents(), direction.value()));
     for (size_t i = 0; i < Dim; ++i) {
-      result->value().get(i) -= particle_position.get(i);
+      result->value().get(i) -= particle_position_velocity.at(0).get(i);
```

Comment:
Question: You only use the position here. Presumably in a future PR you'll use the velocity either here or somewhere else?

Reply:
Yes, for example in #5681